app/vmselect/main.go

```diff
@@ -234,6 +287,11 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 			return true
 		}
 	}
+	p, err := httpserver.ParsePath(path)
```

Comment:
This check can't be moved before handlers `/internal/resetRollupResultCache` and `/admin/tenants`, it requires path to have `/{prefix}/{tenantID}/{suffix}` form.